src/main/java/store/dto/response/ReceiptResponse.java

```diff
@@ -0,0 +1,118 @@
+package store.dto.response;
+
+import java.util.ArrayList;
+import java.util.List;
+import store.model.domain.Receipt.InnerReceipt;
+import store.model.domain.Receipt;
+
+public class ReceiptResponse {
+
+    private final List<InnerReceiptStockResponse> purchaseStocks;
+    private final List<InnerReceiptStockResponse> giftStocks;
+    private final Integer totalPurchaseQuantity;
+    private final Integer totalPurchaseAmount;
+    private final Integer promotionDiscount;
+    private final Integer membershipDiscount;
+    private final Integer finalAmount;
+
+    private ReceiptResponse(List<InnerReceiptStockResponse> purchaseStocks,
+                            List<InnerReceiptStockResponse> giftStocks,
+                            Integer totalPurchaseQuantity,
+                            Integer totalPurchaseAmount,
+                            Integer promotionDiscount,
+                            Integer membershipDiscount,
+                            Integer finalAmount
+    ) {
```

Comment:
완~~전 사소한 내용인데 `(` 열고 개행 한 번 넣어주고 포맷팅하면 조금 더 보기 좋아질 것 같아요!
```suggestion
    private ReceiptResponse(
        List<InnerReceiptStockResponse> purchaseStocks,
        List<InnerReceiptStockResponse> giftStocks,
        Integer totalPurchaseQuantity,
        Integer totalPurchaseAmount,
        Integer promotionDiscount,
        Integer membershipDiscount,
        Integer finalAmount
    ) {
```

Reply:
줄이 너무 길어질까봐 그랬는데, 후자가 더 예쁜 것 같군요 ㅎㅎ 감사합니다!!